engine/src/test/java/com/google/android/fhir/index/ResourceIndexerTest.kt

```diff
@@ -1063,6 +1063,155 @@ class ResourceIndexerTest {
     assertThat(resourceIndices.stringIndices.any { it.name == "address" }).isFalse()
   }
 
+  @Test
+  fun index_duplicateString_deDuplicatesStringIndices() {
```

Comment:
supernit: deduplicate is one word. same below

```suggestion
  fun index_duplicateString_deduplicatesStringIndices() {
```